### PR TITLE
Add editor config and update additional lint rules

### PR DESCRIPTION
**Pull request by rottencandy**
Everyone uses different editors/IDEs which have their own defaults. This creates lots of unimportant changes and inconsistency in commits.

Having default formats specified would make it easier to maintain standard stuff like indentations and quotes and a linter will also catch commonly occurring syntax bugs.

### CODE REVIEW

**Comment by rottencandy** [comment]
@zeners @Kumaravinash9 I've added some editor defaults here, if you like extra options or more settings, we can add them in the config here.

**Comment by Kumaravinash9** [comment]
@rottencandy,  It could be much better if we add pipeline testing to check if the coding is formatted or not.

**Comment by rottencandy** [comment]
> @rottencandy, It could be much better if we add pipeline testing to check if the coding is formatted or not.

@csoler would it be a good idea to add a GitHub action that runs a linter on every PR?

**Review by Kumaravinash9 — CHANGES_REQUESTED** [review]

File: webui-src/.prettierrc [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "arrowParens": "always",
```
Comment (Kumaravinash9) [inline webui-src/.prettierrc:2]:
```javascript
"trailingComma": "es5",
"bracketSpacing": "true",
```

File: webui-src/.eslintrc.json [hunk 2]
```diff
@@ -28,6 +32,30 @@
         "semi": [
             "error",
             "always"
-        ]
+        ],
```
[the remark this answers is not in the document]
Comment (rottencandy) [reply webui-src/.eslintrc.json:35]: Updated.

**Comment by Kumaravinash9** [comment]
@rottencandy, we should add a .prettierignore file for the directory make-src. So, it will not create conflicts whenever we will run prettier.

**Comment by rottencandy** [comment]
Merging the PR, we can run `eslint` locally for now.
And editorconfig should handle common syntax & indentation for all supported editors.

**Pull request merged**